components/Dashboard/Balances.js

```diff
@@ -170,19 +174,70 @@ function Balances() {
             <p>{t("phrase-02")}</p>
             <Link href="/agreements/new">{t("btn-02")}</Link>
           </li>
-          <li className={styles["disabled"]}>
-            <h2>
-              Refer and<br></br> earn
-            </h2>
-            <p>
-              Professionals or contractors that refer the usage of Kyodo, can earn a % of paid value
-              to the protocol.
-            </p>
-            <Link href="#">Get referral link</Link>
+          <li>
+            <h2>{t("call-03")}</h2>
+            <Link href="#" onClick={() => setIsOpen(true)}>
+              {t("btn-03")}
+            </Link>
           </li>
         </ul>
       </section>
       <Payments limit={2} />
+      <Drawer open={isOpen} onClose={() => setIsOpen(false)} className="drawer">
+        <h2>Personal info</h2>
```

Comment:
What is all of this? We already have this stuff inside the onboarding components